.github/workflows/gh-pages.yaml

```diff
@@ -11,24 +11,40 @@ on:
     paths:
       - 'book/**'
 
+env:
+  CARGO_TERM_COLOR: always
+  # Update the language picker in index.hbs to link new languages.
+  LANGUAGES: zh
```

Comment:
not very sure, but seems this env is set to the *whole* gh-pages. So will this cause the whole book to be zh instead of en?

Reply:
It use it in [Comprehensive Rust](https://github.com/google/comprehensive-rust/blob/main/.github/workflows/publish.yml#L22) to maintain a list of languages for which we should publish translations. The variable name is not significant for `mdbook` or anything else — I just added it so that I can decouple having translations in the repository from actually publishing them on our site.